tools/integration_tests/read_large_files/read_large_files_test.go

```diff
@@ -48,3 +50,11 @@ func TestMain(m *testing.M) {
 
 	os.Exit(successCode)
 }
+
+func CreateFileInLocalDiskAndCopyFileFromLocalDiskToMntDir(fileInLocalDisk string, fileInMntDir string, fileSize int, t *testing.T) {
```

Comment:
The function name looks too long. Can we shorten it to something like CreateFileOnDiskAndCopyToMntDir ?